resources/report-table-formatter.js

```diff
@@ -1,13 +1,64 @@
 {
     "pangolin strain (#SNPs)": function format(value) {
+    $(function () {
+        $('[data-toggle="popover"]').popover()
+    })
+
     $(function () {
         $('[data-toggle="tooltip"]').tooltip()
     })
 
     if (value !== "no strain called") {
-        var lineage = value.split(' ')[0];
+        var split = value.split(' ');
+        var lineage = split[0];
         var link = `<a data-toggle="tooltip" data-placement="top" title="Linkout to cov-lineages" href='https://cov-lineages.org/lineages/lineage_${lineage}.html' target='_blank'>${lineage}</a>`;
-        return link + value.split(lineage).pop();
+
+        const table = `<div style="height: 200px; overflow-y: auto; white-space:pre-wrap;"><table class="table">
```

Comment:
Not for this PR, but in a follow-up, I'd like you to replace these implementations here with a [nunjucks](http://mozilla.github.io/nunjucks) based templating.

Reply:
That looks pretty cool and really makes sense to me. All that juggling around with string and then concatenating them all back together feels just "wrong".